apps/store/src/app/switcher/[id]/InsuranceSwitchExpirationDateForm.tsx

```diff
@@ -24,7 +24,7 @@ export default function InsuranceSwitchExpirationDateForm({ id }: Props) {
   const { t } = useTranslation('contractSwitchConfirmationForm')
   const [state, formAction] = useFormState(submitSwitchConfirmation, null)
 
-  console.log({ id })
+  const isSuccessful = state?.messages?.find((message) => message.type === 'success')
```

Comment:
nit:
Since we don't actually need to hold the successful message but only a boolean representing if there's a successful message or not, `some` could be a better fit:

```suggestion
  const isSuccessful = state?.messages?.some((message) => message.type === 'success')
```

Reply:
Great point!